### PR TITLE
Des 48

### PR DESCRIPTION
**Pull request by climberfields**
Created documentation for reindexing files. The documentation is under designsafe.apps.api.tasks.reindexing. I could not find the area to include it in the index. Making PR now, if I have to add it to index I will do so once PR is updated.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Configuration file for the Sphinx documentation builder.
```
Comment (xirdneh) [inline docs/conf.py]: We already have a configuration [in the repo](https://github.com/DesignSafe-CI/portal/tree/master/.docs/source/conf.py)

File: docs/Makefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Minimal makefile for Sphinx documentation
```
Comment (xirdneh) [inline docs/Makefile]: We already have a Make file for this in [the repo](https://github.com/DesignSafe-CI/portal/blob/master/.docs/Makefile)

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.. Indexing_Files documentation master file, created by
```
Comment (xirdneh) [inline docs/index.rst]: We already have this in [the repo](https://github.com/DesignSafe-CI/portal/blob/master/.docs/source/index.rst)

File: docs/make.bat [hunk 4]
```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```
Comment (xirdneh) [inline docs/make.bat]: There isn't a need for a `.bat` file. That's windows specific.

File: .docs/source/designsafe.apps.api.tasks.reindexing.rst [hunk 5]
```diff
@@ -0,0 +1,42 @@
+Reindexing
```
Comment (xirdneh) [inline .docs/source/designsafe.apps.api.tasks.reindexing.rst]: This documentation needs to be as a docstring in the [task](https://github.com/DesignSafe-CI/portal/blob/master/designsafe/apps/api/tasks.py#L23)

**Comment by xirdneh** [comment]
@climberfields The files are still there but those are duplicates.
We already have everything setup in `.docs/`

**Comment by climberfields** [comment]
Should be good to go now

**Comment by jarosenb** [comment]
> @jarosenb is this still applicable for the new indexer?

This is for the old indexer. I'll need to tweak it.

**Pull request closed without merging**